pkg/ovssubnet/common.go

```diff
@@ -137,11 +149,16 @@ func (oc *OvsController) validateServiceNetwork(networkCIDR string) error {
 }
 
 func (oc *OvsController) validateNetworkConfig(clusterNetworkCIDR, serviceNetworkCIDR string, subnetsInUse []string) error {
+	hostIPs, err := net.LookupIP(oc.hostName)
```

Comment:
Would it be better to loop over `net.Interfaces()`? That would let us check multiple interfaces, and would get all IPs on the system, rather than just the ones listed in DNS.